### DISCUSSION TITLE
Cannot run with larger runners

### DISCUSSION THREAD

**Discussion by quom** [community/community#45789] [Actions] [answered]
### Select Topic Area

Bug

### Body

I am having issues using the default larger runners in my workflows. I have checked the following:

- Runners enabled in the Org and shared with the repository
- Non-zero spending limit for actions runners
- Tried using both `runs-on: group: ubuntu-runners` and `runs-on: ubuntu-latest-8-cores` in the workflow job definition.

Under the organisation Runner Groups page I can see the 4 default larger runners with Status: Ready but no jobs are scheduled on these runners.

The job is stuck here:
```
Requested runner group: ubuntu-runners
Job defined at: XXX
Waiting for a runner to pick up this job...
```

Here are the runners in the settings page for the repository:
![image](https://user-images.githubusercontent.com/4928671/215479335-9ad90d0d-5a15-4a0f-9a95-17b4ab2be8b8.png)

Any ideas?

Some updated info:

I adjusted the runner group name and set the runs-on parameter to the group and it scheduled a job on a windows node but not ubuntu. I then removed the windows runners from the group and it would not schedule any jobs on the remaining large linux runners.

**Comment by quom** [comment, marked as answer]
This might be related: https://www.githubstatus.com/incidents/d1l8lxw26y8k

**Comment by taschik** [comment]
The same issue is happening now for us too but the status page is showing no issues. Any ideas?
![Screenshot 2023-08-01 at 1 07 36 PM](https://github.com/community/community/assets/351623/1bc6b6c3-d2eb-4fc0-b8ed-eb2eabddbb5c)

![Screenshot 2023-08-01 at 1 07 23 PM](https://github.com/community/community/assets/351623/d5e7b371-da59-4c2a-983c-e803db3bf93f)

**Reply by matteovivona** [reply]
@taschik How did you solve the problem? I have been in the same situation for a few hours.

**Reply by taschik** [reply]
@matteovivona we don't use the large runners and defaulted to the default runners
